bitnami/matomo/Chart.yaml

```diff
@@ -38,5 +38,4 @@ maintainers:
   url: https://github.com/bitnami/charts
 name: matomo
 sources:
-- https://github.com/bitnami/charts/tree/main/bitnami/matomo
```

Comment:
Why did you remove the sources array?

Reply:
Seems a bad resolved merge, let's update it. Also, CP has published a fix for the VIB/CI job. Let's merge this!